x-pack/platform/plugins/shared/alerting/server/lib/wrap_scoped_cluster_client.ts

```diff
@@ -162,7 +162,10 @@ function getWrappedTransportRequestFn(opts: WrapEsClientOpts) {
     options?: TransportRequestOptions
   ): Promise<TResponse | TransportResult<TResponse, TContext>> {
     // Wrap ES|QL requests with an abort signal
-    if (params.method === 'POST' && params.path === '/_query') {
+    if (
+      (params.method === 'POST' && ['/_query', '/_query/async'].includes(params.path)) ||
+      (params.method === 'GET' && params.path.startsWith('/_query/async'))
+    ) {
```

Comment:
This is really small, but in this wrapper we log the search duration. We may not want to log metrics for async searches bc the query times won't be accurate.

Reply:
We have set wait_for_complete to 4m. So, query duration can be up to 4m., which is quite significant value.
And afterwards, we'll have polling requests that would help to build a whole picture of requests timings